src/transformState.js

```diff
@@ -5,7 +5,30 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  let keys = [];
+  let result = state;
```

Comment:
Result is redundant variable, you can work with state in your code, they both contain link on the same object